cartographer/mapping/internal/constraints/constraint_builder_2d.cc

```diff
@@ -326,6 +329,10 @@ void ConstraintBuilder2D::RegisterMetrics(metrics::FamilyFactory* factory) {
       "Constraint scores built", boundaries);
   kConstraintScoresMetric = scores->Add({{"search_region", "local"}});
   kGlobalConstraintScoresMetric = scores->Add({{"search_region", "global"}});
+  auto* num_matchers = factory->NewGaugeFamily(
+      "constraints_constraint_builder_2d_num_submap_scan_matchers",
```

Comment:
Any reason this is not prefixed with `mapping_` like all other metrics here?

Reply:
no, changed it.